js/app/widgets/lightbox.js

```diff
@@ -275,14 +274,14 @@ const LightboxContainer = new Knowledge.Class({
         this._previous_button = new Gtk.Button({
             halign: Gtk.Align.START,
             valign: Gtk.Align.START,
-            margin: this._ICON_MARGIN,
+            margin: 20,
```

Comment:
I purposefully moved it out of _ICON_MARGIN because they're different values; `this._previous_button` needs margins all around, and `this._close_button` needs different values for `margin-top` and `margin-left`.

Also, yeah, did try to move it to CSS, but something about GTK doesn't seem to like actual CSS margins as much as these explicit margins in the Gtk.Button definition; it would make the next & previous buttons go past the bottom of the screen. Not sure why, so decided it wasn't important enough to spend a ton of time on in this commit.